contents/df_end.md

```diff
@@ -65,16 +72,22 @@ collect(zip(df.name, df.grade_2020))
 However, converting a DataFrame to a Dictionary is only useful when the elements are unique.
 When that is not the case, it is time to `filter` (@sec:filter).
 
-## Filter and Subset {#sec:filter}
+## Filter and Subset {#sec:filter_subset}
+
+There are two ways to remove rows from a DataFrame, one is `filter` (@sec:filter) and another one is `subset` (@sec:subset).
+`filter` was added earlier to `DataFrames.jl`, is more powerful, and more consistent with syntax from Julia base, so that is why we start with discussing `filter`.
+`subset` is a newer, and often more convenient version.
+
+### Filter {#sec:filter}
+
+From this point on, we start to really get to grips with `DataFrames.jl`.
+To this end, we need to learn some verbs, such as `select` and `filter`, but it might be a relieve to know that the general design goal of `DataFrames.jl` is to keep the number of verbs that a user has to learn to a minimum[^verbs].
```

Comment:
Why mention `select` here? Isn't it better to stick with just `filter`. Is the added complexity necessary?

Reply:
Because I needed an example for the word "verbs". Without these examples, I don't think that it's clear what a "verb" means here in this context. I'll merge it now, but you are welcome to bring it up later again of course :)